recipes/pygenometracks/meta.yaml

```diff
@@ -21,13 +21,14 @@ requirements:
   run:
     - python >=3.6
     - numpy >=1.16
-    - matplotlib >=3.0
+    - matplotlib-base =3.1.1
```

Comment:
Requiring this exact version is quite strict. For example, if one would try to install this at the same time as another tool that requires `>=3.1.2`, it wouldn’t work. Is that actually necessary? Or would perhaps something like `>=3.1.1,<3.2` work?

Reply:
We made bad experience concerning the stability of API of matplotlib even for minor changes like 3.0.1 to 3.0.2. Moreover, matplotlib likes to change the size of the images, default colors, default font and font sizes from (minor) update to (minor) update. If this leads to crashes in pyGenomeTracks or that the visualization does not look like it was intended by us. However, the users will blame us for all this. We are simply truly annoyed by this behaviour and fix therefore the version number. We recommend to use pyGenomeTracks in its own environment.

Reply:
Thanks for the explanation!